packages/netlify-cms-widget-list/src/ListControl.js

```diff
@@ -542,7 +573,7 @@ export default class ListControl extends React.Component {
             )}
           >
             <ObjectWidgetTopBar
-              allowAdd={field.get('allow_add', true)}
+              allowAdd={allowAdd}
```

Comment:
Same comment for disappearing button as `remove`

Reply:
see new commit